package.json

```diff
@@ -51,13 +52,11 @@
     "mocha": "^9.0.0",
     "npm-run-all": "^4.1.5",
     "ts-node": "^10.0.0",
-    "typescript": "^4.3.2"
+    "typescript": "^4.3.2",
+    "isomorphic-fetch": "^3.0.0"
```

Comment:
Maybe you want to also install the types for this dep? https://www.npmjs.com/package/@types/isomorphic-fetch

```
npm install --save-dev @types/isomorphic-fetch
```

Also, is this a dev dependancy? I would assume it isn't but i might be wrong.

Reply:
It actually *is* a dev dependency. Moch runs the tests in a node environment without `fetch` available. It's only necessary because 3 tests actually perform real `fetch` requests. We might as well mock `fetch` that would probably be cleaner.